app/src/main/java/org/p2p/wallet/striga/kyc/sdk/StrigaKycSdkListeners.kt

```diff
@@ -0,0 +1,53 @@
+package org.p2p.wallet.striga.kyc.sdk
+
+import com.sumsub.sns.core.SNSActionResult
+import com.sumsub.sns.core.data.listener.SNSEvent
+import com.sumsub.sns.core.data.model.SNSCompletionResult
+import com.sumsub.sns.core.data.model.SNSException
+import com.sumsub.sns.core.data.model.SNSSDKState
+
+/**
+ * Listener to know about errors that occur within the SDK
+ *
+ * [documentation](https://developers.sumsub.com/msdk/android/#on-sdk-errors)
+ */
+typealias StrigaSdkErrorListener = (error: SNSException) -> Unit
```

Comment:
I'm uncertain whether prefixing this with `Striga` is appropriate, given its actual association with SumSub. When we add another provider, this will likely be reused. As alternatives we could consider names such as `KycSdk*` or `SumSub*/Sumsub*`

Reply:
For another provider - we will have another service, that's for sure